executor/show_test.go

```diff
@@ -537,13 +537,9 @@ func (s *testSuite) TestShow2(c *C) {
 	tk.MustQuery("show tables").Check(testkit.Rows("t"))
 	tk.MustQuery("show full tables").Check(testkit.Rows("t BASE TABLE"))
 
-	r, err := tk.Exec("show table status from test like 't'")
+	_, err := tk.Exec("show table status from test like 't'")
 	c.Assert(err, IsNil)
-	row, err := r.Next(context.Background())
```

Comment:
ditto

Reply:
you can rewrite this unit test to:
```go
r := tk.MustQuery("show table status from test like 't'")
r.Check(testkit.Rows(...))
```

Reply:
ok,I will try to fix it.

Reply:
I am not very clear about `r.Check(testkit.Rows())`, can I modify it to resemble the following unit test code?

```
r3, err := tk.Exec("show table status from test like 't'")
	c.Assert(err, IsNil)
	chk := r3.NewChunk()
	err = r3.NextChunk(context.Background(),chk)
	c.Assert(err, IsNil)
	if chk.NumRows() != 0 {
		var idx int
		for rowIdx := 0; rowIdx < chk.NumRows(); rowIdx++ {
			row := chk.GetRow(rowIdx)
			c.Assert(row.GetInt64(0), Equals, int64(idx))
			idx++
		}
	}
```

Reply:
Yes, this is kind of a solution, but some code logical is duplicated with `r.Check(testkit.Rows())`. `MustQuery` and `r.Check(testkit.Rows())` is more convenient. You can see other test cases for examples about how to use `MustQuery`.